package.json

```diff
@@ -13,6 +13,7 @@
     "fastify-error: ~0.2.0"
   ],
   "dependencies": {
+    "content-disposition": "^0.5.3",
```

Comment:
@Julien-Torrent, let's make sure that we install pinned version packages. To do this for NPM, add an `.npmrc` file with `save-exact=true` inside. Then we need to remove the special chars from the deps and reinstall.